libraries/rush-lib/src/logic/base/BaseInstallManager.ts

```diff
@@ -948,6 +948,28 @@ ${gitLfsHookHandling}
         args.push(`--config.auto-install-peers=${autoInstallPeers}`);
       }
 
+      /*
+        If user set minimum-release-age in pnpm-config.json only, use the value in pnpm-config.json
+        If user set minimum-release-age in pnpm-config.json and .npmrc, use the value in pnpm-config.json
+        If user set minimum-release-age in .npmrc only, do nothing, let pnpm handle it
+      */
+      const isMinimumReleaseAgeInNpmrc: boolean = isVariableSetInNpmrcFile(
+        subspace.getSubspaceConfigFolderPath(),
+        'minimum-release-age',
+        this.rushConfiguration.isPnpm
+      );
+
+      const minimumReleaseAge: number | undefined = this.rushConfiguration.pnpmOptions.minimumReleaseAge;
+      if (minimumReleaseAge !== undefined) {
+        if (isMinimumReleaseAgeInNpmrc) {
+          this._terminal.writeWarningLine(
+            `Warning: PNPM's minimum-release-age is specified in both .npmrc and pnpm-config.json. ` +
+              `The value in pnpm-config.json will take precedence.`
+          );
+        }
+        args.push(`--config.minimumReleaseAge=${minimumReleaseAge}`);
```

Comment:
Instead of passing this via command-line, it seems like we should write this to the workspace config file in `common/temp`